types/src/front/api_handlers/public/vaults.ts

```diff
@@ -3,14 +3,12 @@ import * as t from "io-ts";
 import { ContentNodeType } from "../../lib/connectors_api";
 
 export const ContentSchema = t.type({
-  // TODO(DATASOURCE_SID): rename to `dataSourceId`
-  dataSource: t.string,
+  dataSourceId: t.string,
```

Comment:
This will temporarily break for people on the pages when we release... but I guess it's fine.

Reply:
Given not released I think so ? Also this one is not used in practice